skipper_test.go

```diff
@@ -301,3 +305,35 @@ func TestHTTPServerShutdown(t *testing.T) {
 	wg.Wait()
 	time.Sleep(d)
 }
+
+type (
+	customRatelimitSpec   struct{ registry *ratelimit.Registry }
+	customRatelimitFilter struct{}
+)
+
+func (s *customRatelimitSpec) Name() string { return "customRatelimit" }
+func (s *customRatelimitSpec) CreateFilter(config []interface{}) (filters.Filter, error) {
+	log.Infof("Registry: %v", s.registry)
```

Comment:
I guess the log is a left over, right?

Reply:
> I guess the log is a left over, right?

I left it for purpose.

The `Example_ratelimitRegistryBinding` is compiled but not run because there is `// Output:` comment. By adding this comment it is possible to see the output of this log.